builtin/providers/google/resource_google_project.go

```diff
@@ -210,6 +231,18 @@ func resourceGoogleProjectRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("policy_etag", pol.Etag)
 	d.Set("policy_data", string(polBytes))
 
+	// Read the billing account
+	ba, err := config.clientBilling.Projects.GetBillingInfo("projects/" + pid).Do()
+	if err != nil {
+		return fmt.Errorf("Error reading billing account for project %q: %v", "projects/"+pid, err)
+	}
+	if ba.BillingAccountName != "" {
+		_ba := strings.Split(ba.BillingAccountName, "billingAccounts/")
```

Comment:
The way this is written, I don't actually know what the string is expected to be... it could be any one of these:

* projects/{PROJECT}/billingAccounts/{account name}
* billingAccounts/{account name}
* https://www.googlespis.com/projects/{PROJECT}/billingAccounts/{account name}

Honestly, would love some clarification (esp as a code comment) around what `ba.BillingAccountName` is expected to be at this point. And agree `strings.Split` may not be the most semantic function to use.